static/app/views/issueDetails/groupActivityItem.tsx

```diff
@@ -98,10 +98,9 @@ function GroupActivityItem({
   author,
 }: GroupActivityItemProps) {
   const issuesLink = `/organizations/${organization.slug}/issues/`;
-  const hasEscalatingIssuesUi = organization.features.includes('escalating-issues');
 
   function getIgnoredMessage(data: GroupActivitySetIgnored['data']) {
-    const ignoredOrArchived = hasEscalatingIssuesUi ? t('archived') : t('ignored');
+    const ignoredOrArchived = t('archived');
```

Comment:
let's rename for clarity

```suggestion
    const archived = t('archived');
```